runner/src/simulate.rs

```diff
@@ -617,6 +628,17 @@ impl ScenarioState {
                     Err(e) => (CommandResult::Failure(e), None),
                 }
             }
+            Scenario::CASBenchmark => {
+                let ids = self
```

Comment:
Is this just for debugging/understanding things and we'll enforce the limits later?